docs/archive/user/alphaFeatures/Interesting.md

```diff
@@ -6,8 +6,7 @@ The _interesting generation mode_ exists to provide a means of generating smalle
 
 ```
 {
-	"schemaVersion": "0.1",
-	"fields": [
+    "fields": [
```

Comment:
The indentation on this file looks rather strange in GitHub - can this be sorted please